mllib/src/test/scala/org/apache/spark/ml/regression/AFTSurvivalRegressionSuite.scala

```diff
@@ -27,6 +27,9 @@ import org.apache.spark.ml.util.TestingUtils._
 import org.apache.spark.mllib.random.{ExponentialGenerator, WeibullGenerator}
 import org.apache.spark.mllib.util.MLlibTestSparkContext
 import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.functions.{col, lit}
+import org.apache.spark.sql.types.{ByteType, DecimalType, FloatType, IntegerType, LongType,
+  ShortType}
```

Comment:
The style rule is generally to use `_` when you're importing >= 5 things. You can revert it back, thanks!